tests/__snapshots__/PdkOrderTest__it_creates_a_storable_array_with_data_set_carrier_UPS_to_Belgium__1.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "exported": false,
+    "deliveryOptions": {
+      "carrier": {
+        "externalIdentifier": "ups"
+      },
+      "labelAmount": 1,
+      "shipmentOptions": {
+        "insurance": -1,
+        "ageCheck": -1,
+        "return": -1,
+        "hideSender": -1,
+        "largeFormat": -1,
+        "onlyRecipient": -1,
+        "receiptCode": -1,
+        "sameDayDelivery": -1,
+        "signature": -1,
+        "tracked": -1,
+        "collect": -1
+      },
+      "deliveryType": "express",
```

Comment:
Klopt het dat hier ‘express’ komt naar België? Wat niet specifiek genoemd is in de snapshot naam en verder in deze PR niet toegestaan lijkt te worden?

Reply:
Goed gezien! Ik heb er naar gekeken maar voor zover ik zie wordt hier alleen getest of je iets in een array kan veranderen. Er wordt geen validatie toegepast.